app/views/providers/limitations/show.html.erb

```diff
@@ -0,0 +1,35 @@
+<%= page_template(
+      page_title: t('.h1-heading')
+    ) %>
+
+<% if @legal_aid_application.proceeding_types.present? %>
```

Comment:
Not a fan of this. I wonder if we should redirect to the previous page in that case.
With something like this in the controller:
```ruby
    def show
      unless legal_aid_application.proceeding_types.present?
        redirect_to providers_legal_aid_application_proceedings_types_path(legal_aid_application)
      end
    end
```

Reply:
the if block is not required so I have removed it. I had thought that it was possible to create an application without a proceeding type.